### PR TITLE
fix(*): schema desc transformation in plugin forms

### PR DESCRIPTION
**Pull request by sumimakito**
# Summary

This PR makes the plugin form use descriptions from backend schemas when descriptions are not defined in frontend schemas. It also fixes some current issues with tooltips.

### Tooltips for _schema_ fields

Example: `request-validator`

|Before|After|
|-|-|
|Screenshot 2023-12-20 at 13 58 47|Screenshot 2023-12-20 at 14 00 12|

### Tooltips for nested fields

Example: `rate-limiting-advanced`

|Before|After|
|-|-|
|Screenshot 2023-12-20 at 13 59 12|Screenshot 2023-12-20 at 13 59 44|

### Missing tooltips

Example: `jwt` (some descriptions are missing from Konnect API but available in the Gateway Admin API)

|Before|After|
|-|-|
|Screenshot 2023-12-20 at 13 57 47|Screenshot 2023-12-20 at 14 00 47|

### Outdated tooltips

Example: `kafka-upstream`

Screenshot 2023-12-20 at 13 57 31

KAG-3347

### CODE REVIEW

**Comment by sumimakito** [comment]
Note: although it seems that we are not using `packages/core/forms/src/forms/ACMEForm.vue` by default anymore, I just updated that component in case it is used

**Review by kaiarrowood — COMMENTED** [review]
This all makes sense to me, but I did not functional test 🚀

**Review by kaiarrowood — APPROVED** [review]

**Review by adamdehaven — CHANGES_REQUESTED** [review]
@sumimakito, @hl782 would like to test this before merge. I'm pulling the package into [this `khcp-ui` PR](https://github.com/Kong/khcp-ui/pull/7336); and blocking merge until he signs off (no actual issues).

**Review by adamdehaven — APPROVED** [review]
Verified by @hl782

**Pull request merged**
